extensions/wikia/AdEngine/AdEngine2Hooks.class.php

```diff
@@ -86,6 +86,7 @@ public static function onInstantGlobalsGetVariables( array &$vars ) {
 		$vars[] = 'wgAdDriverMegaAdUnitBuilderForFVCountries';
 		$vars[] = 'wgAdDriverMoatTrackingForFeaturedVideoAdCountries';
 		$vars[] = 'wgAdDriverMoatTrackingForFeaturedVideoAdSampling';
+		$vars[] = 'wgAdDriverMoatTrackingForFeaturedVideoRvS1Countries';
```

Comment:
What about `wgAdDriverMoatTrackingAdditionalParametersCountries`?

Reply:
We have separate moat wgVars for Porvata, so what about `wgAdDriverMoatTrackingForFeaturedVideoAdditionalParamsCountries` :D?

Reply:
I like this one: wgAdDriverMoatTrackingForFeaturedVideoAdditionalParamsCountries
😆

Reply:
Staying at wgAdDriverMoatTrackingForFeaturedVideoAdditionalParamsCountries